golang/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM dockerfile/ubuntu
+
+MAINTAINER Filip Gospodinov "filip@monetas.net"
+# use bash for docker RUN commands; we want associative arrays
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# use custom GOPATH such that it does not conlfict with GOPATHs that
+# live inside the host's $HOME which may be bind mounted into docker
+ENV GOPATH $HOME/go_monetas
+ENV GOROOT $HOME/go_dist/go
+ENV PATH $GOROOT/bin:$GOPATH/bin:$PATH
```

Comment:
It also just occurred to me that this may not work at all when mounting the host's $HOME, because the user may have, say, a .bashrc that overrides this?